pages/become_a_member.vue

```diff
@@ -0,0 +1,33 @@
+<template>
+    <div class="max-w-7xl mx-auto spx-6 px-6 lg:px-8">
+        <h1 class="text-2xl font-bold py-4">Become a Member</h1>
+        <p class="py-2">
+            Thank you for your interest in becoming a DES member! Firstly we would like to point out that <strong> you do not need to be a member to use the DES Slack, attend meetups and events, or to join the monthly society meeting and AGM. </strong> Membership is only required if you wish to vote, or use DES equipment and resources (like our YouTube channel or streaming tools.)
```

Comment:
So it seems to not want me to suggest changes to multiple lines, so hopefully this still makes sense...

```suggestion
            Thank you for your interest in becoming a DES member! Firstly we would like to point out that <strong> you do not need to be a member to use the DES Slack, attend meetups and events, or to join the monthly society meeting and AGM. </strong>
            
            Any member of the public is welcome to:

            * Join the Edmonton developer community            
            * Chat on our Slack
            * Attend DES supported meetups and events
            * Come out to DES monthly planning meetings
            * Participate in Working Groups
            
            <strong>Membership is only required if:</strong>
            
            * you wish to vote on society business
            * borrow DES equipment (cameras, mics, etc.)
            * use the DES resources (like our YouTube channel or streaming tools)
            * start an event or group officially supported by DES
            * use the DES name or branding
            * champion a Working Group
            * you want to give us $20/year to support the work we do (thank you!)
```

I'm thinking re-stating that you don't need to be a member in text and in bullets, along with a specific list of the times you do might help people understand the difference.